Memorize/Screens/ThemeEditor/ThemeEditorView.swift

```diff
@@ -3,6 +3,22 @@ import SwiftUI
 struct ThemeEditorView: View {
     var theme: ThemeEditorViewModel
     @State private var emojisToAdd = ""
+    @State var pairsValue: String
+
+    init(theme: ThemeEditorViewModel, pairsValue: ThemeModel.NumOfPairs) {
+        var num: String {
+            if pairsValue == .all {
+                return "All"
+            } else if pairsValue == .random {
+                return "Random"
+            } else {
+                return "Explicit"
+            }
+        }
+
+        self.theme = theme
+        self.pairsValue = num
```

Comment:
`num`? what is `num`? )

Reply:
Yes, sorry. I tried different approaches and forgot to rename it